roles/openshift_web_console/templates/console-smoke-test-pod.yaml.j2

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: openshift-web-console-smoke-test-pod
+  labels:
+    app: openshift-web-console-smoke-test
+  annotations:
+    prometheus.io/scrape: "true"
+    prometheus.io/scheme: http
+spec:
+  restartPolicy: Never
+  containers:
+  - name: openshift-web-console-smoke-test
+    image: jhadvig/origin-web-console-smoke-test:latest
+    ports:
+    - containerPort: 3000
+      protocol: TCP
+    imagePullPolicy: IfNotPresent
+    env:
+    - name: CONSOLE_URL
+      value: {{ openshift.master.public_console_url }}
+    - name: TEST_INTERVAL
```

Comment:
Maybe `TEST_INTERVAL_MINUTES` so the unit is clear.

Reply:
Even better, accept a `time.Duration` string 😁

Reply:
@ironcladlou our tests actually run in `js`, not `go`, though we we are discussing what to do with this.  Is it standardized across all the tests as something consistent we should follow?